app/controllers/application_controller.rb

```diff
@@ -9,7 +9,7 @@ class ApplicationController < ActionController::Base
   rescue_from ActiveRecord::RecordNotFound, with: :render_not_found
   rescue_from ActionController::InvalidAuthenticityToken, with: :render_forbidden
   rescue_from ActionController::UnpermittedParameters, with: :render_bad_request
-  rescue_from Pundit::NotAuthorizedError, with: :render_forbidden
+  rescue_from(Pundit::NotAuthorizedError) {|_| render_forbidden }
```

Comment:
It purposely doesn't pass the error so that it's not rendered, I thought this was better than checking the error class in every call to render_forbidden.